vofs/tox.ini

```diff
@@ -5,18 +5,21 @@ name = vofs
 
 [tox]
 envlist =
-    py{27,34,35,36,37,38,39}
+    py{27,34,35,36,37,38,39,310}
 requires =
     pip >= 19.3.1
 
 [testenv]
 description = run tests
 changedir = {toxinidir}
 passenv = HOME
-deps = -rdev_requirements.txt
+deps =
+    -e ../vos
```

Comment:
Should the `vos` dependency be for a specific version, because `vos` isn't going to have Python2 support going forward?

Reply:
Fixed